### PR TITLE
Fixing test infrastructure

### PR DESCRIPTION
**Pull request by vlasenkoalexey**
This should take care of #251 
The actual change is in gcsfs/tests/utils.py, but all replay files had to be regenerated to work with new GCS request format.

### CODE REVIEW

**Comment by vlasenkoalexey** [comment]
Tests are passing when I run on my machine under docker, but failing in travis. Can't figure out what's wrong, will take another look tomorrow.

**Comment by vlasenkoalexey** [comment]
Should also take care of #231

File: gcsfs/tests/test_core.py [hunk 1]
```diff
@@ -702,12 +702,12 @@ def test_writable():
 def test_merge():
     with gcs_maker() as gcs:
         with gcs.open(a, "wb") as f:
-            f.write(b"a" * 100)
+            f.write(b"a" * 10)
```
Comment (martindurant) [inline gcsfs/tests/test_core.py:705]: Why were these data sizes changed?

Comment (vlasenkoalexey) [reply gcsfs/tests/test_core.py:705]: test was consistently failing with rate exceeded error when executed under default user authentication. As far as I understand, actual value doesn't matter that much.

Comment (martindurant) [reply gcsfs/tests/test_core.py:705]: Indeed it doesn't matter for this test, but it is worrying nevertheless. There should be the same number of requests either way, one would think... Never mind!

**Pull request merged**